partner_contact_company/models/partner.py

```diff
@@ -56,7 +62,7 @@ def name_get(self):
                 if ctx.get('show_address'):
                     name = name + "\n" + self._display_address(
                         record, without_company=True)
-                name = name.replace('\n\n', '\n')
-                name = name.replace('\n\n', '\n')
+                if name:
+                    name = name.replace('\n\n', '\n')
```

Comment:
In fact we need to review that code, the purpose of this "ugly" duplicated line was to support the case where you have 4 useless "\n" and to only have "one" at the end
maybe we should do a loop
```
while "\n\n" in name:
   name = name.replace('\n\n', '\n')
```